### PR TITLE
(GH-13) cleaned up naming.

### LINKED ISSUE #13: Cleanup naming
Remove prefixes from variables, fields and argument names

### PR DESCRIPTION
**Pull request by stefan-lindegger**
Cleaned up naming.
Fixes #13

### CODE REVIEW

File: src/BBT.Maybe.Tests/MaybeStructTests.cs [hunk 1]
```diff
@@ -13,29 +13,29 @@ public sealed class TheSomeStructMethod
...
-                var lReferencingStruct = default(BaseStruct);
-                lReferencedStruct.ReferencingStruct = lReferencingStruct;
+                var referencedStruct = default(ReferencedStruct);
+                var referencingStruct = default(BaseStruct);
+                referencedStruct.ReferencingStruct = referencingStruct;
 
                 // Act
-                var lMaybeReferencingStruct = Maybe.SomeStruct<ReferencedStruct>(lReferencedStruct)
+                var maybeReferencingStruct = Maybe.SomeStruct<ReferencedStruct>(referencedStruct)
                     .Some<BaseStruct>(aX => aX.ReferencingStruct);
```
Comment (jokay) [inline src/BBT.Maybe.Tests/MaybeStructTests.cs]: You may want to change the naming for `aX` too?

There are some more occurences in the PR.

File: src/BBT.Maybe/MaybeStruct.cs [hunk 2]
```diff
@@ -15,84 +15,84 @@
...
         /// </summary>
-        /// <param name="aA">Maybe to compare.</param>
-        /// <param name="aB">Maybe to compare.</param>
-        public static bool operator !=(MaybeStruct<T> aA, MaybeStruct<T> aB)
+        /// <param name="a">Maybe to compare.</param>
+        /// <param name="b">Maybe to compare.</param>
+        public static bool operator !=(MaybeStruct<T> a, MaybeStruct<T> b)
         {
-            return !(aA == aB);
+            return !(a == b);
```
Comment (jokay) [inline src/BBT.Maybe/MaybeStruct.cs]: `return !(a == b);` may be simplified by `return a != b;`

File: src/BBT.Maybe/NoneCase.cs [hunk 3]
```diff
@@ -7,83 +7,83 @@
...
         /// </summary>
-        /// <param name="aA">Maybe to compare.</param>
-        /// <param name="aB">Maybe to compare.</param>
-        public static bool operator !=(NoneCase aA, NoneCase aB)
+        /// <param name="a">Maybe to compare.</param>
+        /// <param name="b">Maybe to compare.</param>
+        public static bool operator !=(NoneCase a, NoneCase b)
         {
-            return !(aA == aB);
+            return !(a == b);
```
Comment (jokay) [inline src/BBT.Maybe/NoneCase.cs]: `return !(a == b);` may be simplified by `return a != b;`

File: src/BBT.Maybe/Maybe.cs [hunk 4]
```diff
@@ -15,125 +15,125 @@
...
         /// </summary>
-        /// <param name="aA">Maybe to compare.</param>
-        /// <param name="aB">Maybe to compare.</param>
-        public static bool operator !=(Maybe<T> aA, Maybe<T> aB)
+        /// <param name="a">Maybe to compare.</param>
+        /// <param name="b">Maybe to compare.</param>
+        public static bool operator !=(Maybe<T> a, Maybe<T> b)
         {
-            return !(aA == aB);
+            return !(a == b);
```
Comment (jokay) [inline src/BBT.Maybe/Maybe.cs]: `return !(a == b);` may be simplified by `return a != b;`

**Review by pascalberger — APPROVED** [review]
LGTM, beside the comments @x-jokay mentioned. 

@stefan-lindegger OK for you if this will be part of a release after 1.0.0? Otherwise PR needs to go into the `release/1.0.0` branch.

**Comment by stefan-lindegger** [comment]
@pascalberger yes thats OK. I'll fix the issues in the subsequent release.

File: src/BBT.Maybe.Tests/MaybeStructTests.cs (hunk 1 above)
Comment (stefan-lindegger) [reply src/BBT.Maybe.Tests/MaybeStructTests.cs]: Done

File: src/BBT.Maybe/NoneCase.cs (hunk 3 above)
Comment (stefan-lindegger) [reply src/BBT.Maybe/NoneCase.cs]: Done.

File: src/BBT.Maybe/MaybeStruct.cs (hunk 2 above)
Comment (stefan-lindegger) [reply src/BBT.Maybe/MaybeStruct.cs]: Done.

File: src/BBT.Maybe/Maybe.cs (hunk 4 above)
Comment (stefan-lindegger) [reply src/BBT.Maybe/Maybe.cs]: Done.

**Pull request merged**